### ISSUE TITLE
Validação de tipo de arquivo

### ISSUE THREAD

**Issue by rhandrade** [rhandrade/tray-theme#10]
### Descrição 
Como usuário do CLI gostaria que a ferramenta verificasse o tipo do arquivo a ser enviado e comunique caso ele não seja compatível.

### Motivo
Ao desenvolvermos um tema estamos constantemente trabalhando com vários tipos de arquivos e podemos não nos atentar ao tipo do arquivo no momento em que criamos. Sem uma validação disso podemos desprender tempo averiguando algo que não é um bug ou problema, e sim uma limitação.

**Comment by leolopes** [comment]
Imagino que a ideia seria validar o tipo de arquivo (e depois talvez até a pasta) para identificar se ele pode ou não ser subido ao servidor, em vez de deixar o servidor testar isso, correto?

**Comment by rhandrade** [comment]
Exatamente @leolopes. As pastas é até que bem fácil mapear pois a Tray permite que seja adicionado novas pastas somente na `css` e `js`. Os tipos de arquivos ai iria validar de acordo com o atual do opencode_theme e verificar possíveis tipos que ocorrem erros, como o caso de arquivos `.scss`

**Comment by leolopes** [comment]
Entendido, acho que vou tentar dar andamento a este.

**Comment by leolopes** [comment]
Eu fiz um esboço inicial dessa melhoria, e estou aprimorando.

Ainda assim, me questiono: se evitarmos a todo custo receber respostas de erro da Tray, não vamos saber quando houver alguma mudança de regra na API.

Será que deveria haver um "modo desenvolvedor", para deixarmos o arquivo subir e ser rejeitado? Assim aprenderemos sempre sobre as mudanças de regra.

**Comment by rhandrade** [comment]
@leolopes Bom questionamento... Acho que podemos usar um modo debug, uma opção extra no arquivo de configurações que se habilitar o modo debug tudo quer é feito gera um log em um arquivo que sempre será ignorado pelo CLI, algo como `.debug.log`.

**Comment by leolopes** [comment]
@rhandrade, interessante isso do arquivo de log, pois assim não polui os logs do prompt de comando.
Quem sabe colocamos num milestone mais para frente?

**Comment by rhandrade** [comment]
@leolopes Sim... Vou já gerar outra issue, atrelada a essa aqui para pensar nessa questão do arquivo de log.

**Comment by rhandrade** [comment]
Issue fechado, pois feature foi implementada.
Versão 1.1.0 liberada! Vlw pessoal 🎉

**Comment by leolopes** [comment]
Boaaa, já vou começar a usar :)
